packages/data-store/src/migrations/3.createPrivateKeyStorage.ts

```diff
@@ -11,17 +11,21 @@ const debug = Debug('veramo:data-store:migrate-private-keys')
  */
 export class CreatePrivateKeyStorage1629293428674 implements MigrationInterface {
   async up(queryRunner: QueryRunner): Promise<void> {
+    function getTable(givenName: string): Table {
+      const entityMetadatas = queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName);
+      return Table.create(entityMetadatas!, queryRunner.connection.driver);
+    }
     function getTableName(givenName: string): string {
       return (
-        queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName)?.tableName ||
-        givenName
+          queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName)?.tableName ||
+          givenName
       )
     }
     // 1.create new table
     debug(`creating new private-key table`)
     await queryRunner.createTable(
       new Table({
-        name: getTableName('private-key'),
+        name: getTable('private-key').name,
```

Comment:
This is indeed what I would expect in other places, where you now also have the getTableName next to getTable. The whole getTableName is what was going wrong in the first place